fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexScrubber.java

```diff
@@ -905,18 +366,16 @@ public Builder setScrubbingPolicy(@Nonnull final ScrubbingPolicy.Builder scrubbi
          */
         public OnlineIndexScrubber build() {
             validate();
-            OnlineIndexer.Config conf = new OnlineIndexer.Config(limit, limit, maxRetries, recordsPerSecond,
-                    progressLogIntervalMillis, increaseLimitAfter, maxWriteLimitBytes,
-                    OnlineIndexer.Config.UNLIMITED_TIME, OnlineIndexer.Config.UNLIMITED_TIME);
```

Comment:
Note that in the old code, it always set `UNLIMITED_TIME` for both the overall time limit and the per-transaction time limit. The new default is that the build gets `UNLIMITED_TIME` as before but each transaction only gets 4 seconds, just like with index builds.